src/main/java/bio/terra/cli/businessobject/Context.java

```diff
@@ -193,7 +201,11 @@ public static void setUser(User user) {
   }
 
   public static Optional<Workspace> getWorkspace() {
-    return Optional.ofNullable(currentWorkspace);
+    if (useOverrideWorkspace) {
+      return Optional.of(overrideWorkspace);
```

Comment:
Agree, it seems like this could be null if someone called `useOverrideWorkspace()` without a subsequent call to `setWorkspace()`?  Alternatively, if the latter is always meant to follow the former, maybe it should be a single call?

Reply:
- Made it `ofNullable` just in case. This should never happen, as you say, but might give us a better error message if it does.
- Also moved the `Workspace.load` call into the `Context.useOverrideWorkspace` method, so that these two method calls will always happen together.